src/components/Home.js

```diff
@@ -18,10 +40,11 @@ const Home = () => {
             crime statistics.
           </p>
           <img
-            className='absolute right-0'
+            className='absolute right-0 cursor-pointer'
             style={{ bottom: '-30px' }}
             src='/img/arrow-down-circle.svg'
             alt='more information below'
+            onClick={scrollToBottom}
```

Comment:
can we simplify this to just be a link with an href that targets a section with an `id` that it takes the user to?